dev_ops/sql/Tables/CreateContracts.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE contracts (
+    id BIGSERIAL PRIMARY KEY,
+	proposer_id BIGINT REFERENCES users (id),
+	streamer_id BIGINT REFERENCES users (id),
+	game TEXT NOT NULL,
```

Comment:
Isnt TEXT like a massivenumber of characters, like a whole books worth of characters? I think a varchar of like 128 is more than enough space to store game name

Reply:
Text is like varchar but it doesn't specify the upper bound. We could probably cap it, doesn't matter to me.

https://www.postgresql.org/docs/9.1/static/datatype-character.html

Reply:
I will change this to VARCHAR(128)